examples/basic/pd-micro-service-cluster.yaml

```diff
@@ -17,8 +17,8 @@ spec:
     image: alpine:3.16.0
   pd:
     # TODO: use a stable version
-    baseImage: pingcap/pd
-    version: nightly
+    baseImage: hub-new.pingcap.net/orchestration/pd
+    version: v8.0.0
     maxFailoverCount: 0
```

Comment:
As https://github.com/tikv/pd/pull/7819 changed pdms interface, we need to update it when pd 8.0.0 upgrade